dashboard/app/models/teacher_profile.rb

```diff
@@ -20,4 +20,42 @@
 #
 
 class TeacherProfile < ActiveRecord::Base
+  include SerializedProperties
+  # teals: True if the teacher was a TEALS teacher.
+  serialized_attrs %w(teals)
+
+  COURSES = [
+    CSD = 'csd'.freeze,
+    CSF = 'csf'.freeze,
+    CSP = 'csp'.freeze,
+    ECS = 'ecs'.freeze
+  ].freeze
+  # TODO(asher): Change the DB to add a unique index on
+  # (studio_person_id, course).
+  validates_inclusion_of :course, in: COURSES
+
+  YEARS = [
+    YEAR_2013_2014 = '2013-2014'.freeze,
+    YEAR_2014_2015 = '2014-2015'.freeze,
+    YEAR_2015_2016 = '2015-2016'.freeze,
+    YEAR_2016_2017 = '2016-2017'.freeze,
+  ].freeze
+  NO_PD = 'no_pd'.freeze
+  validates_inclusion_of :pd_year, in: YEARS + [NO_PD], allow_nil: true
```

Comment:
I just mean, given how easy it's going to be to forget that step, how obvious is it going to be that this needs to be done? And how easily might we end up in a situation in which "2017-2018" is available on production but is failing validation?